src/BuildIt.Forms/BuildIt.Forms.Controls/CameraPreviewControl.xaml.cs

```diff
@@ -297,6 +283,11 @@ public async Task<IReadOnlyList<ICamera>> RetrieveCamerasAsync()
             return await RetrieveCamerasFunc();
         }
 
+        /// <summary>
+        /// TODO Add summary
```

Comment:
TODO

Reply:
I can definitely add a summary for this method, but I don't think I'm the best person to do so, as I don't fully understand what's this method purpose. I added the summary comments just to clear the  stylecop warning